pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -339,60 +336,23 @@ public void testStringLiteral() {
     Assert.assertEquals(groupbyList.get(1).getFunctionCall().getOperands().get(1).getIdentifier().getName(), "FOO");
   }
 
-  @Test
-  public void testConverter()
```

Comment:
While it makes sense to remove the logging, here's a prior conversation on the intention for this test.

https://github.com/apache/incubator-pinot/pull/4387#discussion_r300516547